docs/user_guide/client_api.rst

```diff
@@ -0,0 +1,86 @@
+##########
```

Comment:
above add `.. _client_api:` so page can be referenced

Reply:
thanks, added